docs/transparent/linux.rst

```diff
@@ -12,15 +12,15 @@ achieve transparent mode.
 
     >>> sysctl -w net.ipv4.ip_forward=1
 
-    You may also want to consider enabling this permanently in ``/etc/sysctl.conf``.
+    You may also want to consider enabling this permanently in ``/etc/sysctl.conf`` or newly created ``/etc/sysctl.d/mitmproxy.conf ``.
 
  3. If your target machine is on the same physical network and you configured it to use a custom
     gateway, disable ICMP redirects:
 
-    >>> echo 0 | sudo tee /proc/sys/net/ipv4/conf/*/send_redirects
+    >>> sysctl -w net.ipv4.conf.all.accept_redirects=0
```

Comment:
@jimhigson: Any particular reason why we are disabling `accept_redirects` here? My understanding would be that we just need to not send them.